### PR TITLE
Entity editor and creator components

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-registry/issues/1 |
| Dependencies | https://github.com/ripe-tech/ripe-components-vue/pull/378 |
| Decisions | Created `entity-creator` and `entity-editor`|
| Animated GIF |  **Usage example in ripe-white-admin-ui:**<br>contex-edit_refactor<br>contex-create_refactor |

### CODE REVIEW

**Review by gcandal — CHANGES_REQUESTED** [review]
Make the changes required for https://github.com/ripe-tech/ripe-registry-ui/pull/16/files/04638052dc5740b8a5eec8995d21105460457f85#r498173503

File: vue/components/ui/organisms/form/form.vue [hunk 1]
```diff
@@ -102,12 +102,14 @@
                 v-bind:accept="Boolean(onSave)"
                 v-bind:reject-button-props="{
                     text: 'Discard',
+                    small: true,
```
Comment (gcandal) [inline vue/components/ui/organisms/form/form.vue:105]: Move these changes to https://github.com/ripe-tech/ripe-components-vue/pull/378

**Review by gcandal — CHANGES_REQUESTED** [review]

**Review by gcandal — APPROVED** [review]

**Comment by joamag** [comment]
@gcandal reassigning to you

**Comment by gcandal** [comment]
@joamag this seems like a pre-requisite for the new Twitch milestone.

**Comment by joamag** [comment]
Yeah we must close this one once possible

**Comment by gcandal** [comment]
The diff is messed up, re-opening the PR.

**Pull request closed without merging**